testfile

```diff
@@ -0,0 +1 @@
+test
```

Comment:
The lack of newline here is not really unix friendly :smile: 

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file